scripts/dev_env.sh

```diff
@@ -2,7 +2,7 @@
 
 # Source this file to setup development environment
 
-WD=$(readlink -f $(dirname "$0"))
+WD=$(greadlink -f $(dirname "$0"))
```

Comment:
Please revert this change. If you need to polyfill readlink you can use an alias or script in your path that defers to `greadlink`.